src/core/plugins/spec/reducers.js

```diff
@@ -86,7 +91,7 @@ export default {
     let newState = state.setIn( [ "responses", path, method ], fromJSOrdered(result) )
 
     // ImmutableJS messes up Blob. Needs to reset its value.
-    if (res.data instanceof win.Blob) {
+    if (win.Blob && res.data instanceof win.Blob) {
```

Comment:
Test suite was throwing an error:

`Right-hand side of 'instanceof' is not an object`